dist/obs-bundled-gems.spec

```diff
@@ -105,18 +127,20 @@ bundle config build.ffi --enable-system-libffi
 bundle config build.nokogiri --use-system-libraries
 bundle config build.sassc --disable-march-tune-native
 
-%install
```

Comment:
You accidentally deleted the `%install` section

Reply:
No, not accidentally.

Otherwise the built gems failed.